docgen/src/guide/Refining_hits.md

```diff
@@ -0,0 +1,552 @@
+---
+title: Refining within hits
+mainTitle: Guides
+layout: main.pug
+category: guide
+navWeight: # idk what to put here 🚨
+---
+
+## Introduction
+
+In some interfaces, you want to have buttons _inside_ your Hits that will change the refinement of the current search, rather than as standalone components. Since we can't replicate this pattern with widgets, we will need to go with Refinementlists that don't render anything. Please note that this is an advanced tutorial and requires moderate knowledge of how React and React InstantSearch work. Our final result will look something like this, let's get started!
```

Comment:
Refinementlists => RefinementLists

Mention Virtual Widgets and link to the guide

In the end we should have a clear prerequisites list.